FEATURES.md

```diff
@@ -6,3 +6,22 @@ To close Helm-dashboard, click on the button in the rightmost corner of the scre
 # Multicluster
 If you want to switch to a different cluster, simply click on the corresponding cluster as shown in the figure. [Click here](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) to learn how to access multiple clusters.
 ![Multicluster_screenshot](images/screenshot_multicluster.png)
+
+# Repository
+Essentially, a repository is a location where charts are gathered and can be shared. If you want to learn more about repositories, [click here](https://helm.sh/docs/topics/chart_repository/). You can find the repository in the home section, as depicted in the figure.
+![Repository3](images/screenshot_repository3.png)
+
+You can add the repository by clicking on 'Add Repository', as shown in the figure.
+![Repository](images/screenshot_repository.png)
+
+After completing that step, enter the following data: the repository name and its URL. You can also add the username and password, although this is optional.
+![Repository2](images/screenshot_repository2.png)
+
+Updating means refreshing your repository. You can update your repository as shown in the figure.
+![Repository4](images/screenshot_repository4.png)
+
+If you want to remove your repository from the Helm dashboard, click on the 'Remove' button as shown in the figure.
+![Repository5](images/screenshot_repository5.png)
+
+If you want to install a particular chart from the list of available charts, use the filter as shown in the figure.
```

Comment:
Now, let's improve this section, it is a bit misleading now. To install the chart, you need to hove the line in table, then the "Install" button will appear. Clicking on this button will trigger the install.

The "filter" is just a way to find the desired chart quicker in the table.